java/arcs/core/storage/handle/CollectionHandle.kt

```diff
@@ -20,13 +20,16 @@ import arcs.core.storage.ActivationFactory
 import arcs.core.storage.Dereferencer
 import arcs.core.storage.Handle
 import arcs.core.storage.StorageProxy
+import arcs.core.storage.Store
 import arcs.core.storage.StoreOptions
 import arcs.core.util.Time
 
 /** These typealiases are defined to clean up the class declaration below. */
 typealias CollectionData<T> = CrdtSet.Data<T>
 typealias CollectionOp<T> = CrdtSet.IOperation<T>
 typealias CollectionStoreOptions<T> = StoreOptions<CollectionData<T>, CollectionOp<T>, Set<T>>
+typealias CollectionStore<T> = Store<CollectionData<T>, CollectionOp<T>, Set<T>>
+typealias CollectionActiveStore<T> = Store<CollectionData<T>, CollectionOp<T>, Set<T>>
```

Comment:
Is this needed?

Reply:
nope, was from an old attempt at something more type-y, will remove